src/sync/sync_manager.cpp

```diff
@@ -415,7 +415,11 @@ std::string SyncManager::path_for_realm(const SyncUser& user, const std::string&
     if (user.token_type() == SyncUser::TokenType::Normal)
         user_info = SyncUserIdentifier{ user.identity(), user.server_url() };
 
-    return m_file_manager->path(user_local_identity, raw_realm_url, std::move(user_info));
+    // FIXME: check for forward slash
+    // FIXME: how to derive unique identifier?
```

Comment:
A random string (like a UUID or something) should do the trick, if persisted and reused.

Ideally it would be possible to override it in the Config too, since there are use cases for that (shared partial views).

Reply:
My thought for the unique identifier was to use a UUID that we then store in the sync metadata Realm.

Making the unique identifier configurable seems like something we can punt on for now in the interests of minimizing the amount of work required from each binding. It shouldn't be hard to add after the fact.

Reply:
If we can use a per-user UUID (e.g. each {user, Realm} tuple only has one partially-synced Realm aside from custom shared partial views), we already generate a local UUID for this purpose and could use that.